drivers/ethernet/Kconfig.stm32_hal

```diff
@@ -79,5 +79,14 @@ config ETH_STM32_HAL_MAC5
 	  This is the byte 5 of the MAC address.
```

Comment:
Thank you for your reply，If you have the opportunity to experiment on hardware it's best，The latest Ethernet driver of stm32Cube does not support PTP, and I don't think st will be added recently, That's better I add the code from the ext/hal/  to driver/ethernet/eth_stm32_hal.c?

Reply:
If we want this feature to be there and as it might take long time before HAL will have PTP support, I suppose one option would be to isolate the required PTP related changes for example to eth_stm32_ptp.c file or similar. But I will leave the final decision about this to @erwango

Reply:
@Jxudong 

> That's better I add the code from the ext/hal/ to driver/ethernet/eth_stm32_hal.c?

Yes, it would be preferred. And I think @jukkar 's proposal makes sense as well (isolate changes in eth_stm32_ptp.c), if doable.